core/email_api.php

```diff
@@ -1549,11 +1549,13 @@ function email_format_bug_message( array $p_visible_bug_data ) {
 			$t_time_tracking = '';
 		}
 
+		$t_access_level_string = ' - ';
 		if( user_exists( $t_bugnote->reporter_id ) ) {
 			$t_access_level = access_get_project_level( $p_visible_bug_data['email_project_id'], $t_bugnote->reporter_id );
-			$t_access_level_string = ' (' . get_enum_element( 'access_levels', $t_access_level ) . ') - ';
-		} else {
-			$t_access_level_string = '';
+			# Only display access level when higher than 0 (ANYBODY)
```

Comment:
> I wonder if we should show a string that user as no access

Thas is a nice idea. I'd say, place a notation next to the user name, that can be a string as you suggested, or a colored asterisk with a popup text (for a leaner ui)

This notice should be showed on most user strings printed in the page: user for notes, reporter user, etc.

In my instance i have a plugin that prints a warning in the bug page when the reporter has no access, or has access but cant write notes. This is needed for me because bugs are moved between private projects, and you may not notice that the reporter will no longer have access in some cases.

Can we open a separate feature request?